python/ray/dashboard/client/src/service/log.ts

```diff
@@ -51,12 +56,14 @@ export const getStateApiDownloadLogUrl = ({
       : []),
     ...(suffix !== undefined ? [`suffix=${encodeURIComponent(suffix)}`] : []),
     `lines=${maxLines}`,
+    `format=${format}`,
   ];
 
   return `api/v0/logs/file?${variables.join("&")}`;
 };
 
 export const getStateApiLog = async (props: StateApiLogInput) => {
+  props.format = "leading_1";
   const url = getStateApiDownloadLogUrl(props);
```

Comment:
```suggestion
  const url = getStateApiDownloadLogUrl({ ...props, format: "leading_1" });
```

To avoid mutating the passed in variable leading to hard to debug bugs later if some logic changes.